exercises/practice/goldbach/goldbach_test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from goldbach import goldbach
+
+def tupleEqualNoOrder(t1: tuple, t2: tuple) -> bool:
+    """Check if two two-element tuples have the same elements."""
+    assert len(t1) == 2
+    assert len(t2) == 2
+    if(t1[0] == t2[0]):
+        return t1[1] == t2[1]
+    else:
+        return t1[0] == t2[1] and t1[1] == t2[0]
+
+class GoldbachTester(unittest.TestCase):
+    def testall(self):
+        cases = [(4, (2, 2)), (6, (3, 3)), (8, (5, 3)), (12, (5, 7))]
+        for (even, primes) in cases:
```

Comment:
Do you think it would be beneficial to the user if we had separate test cases instead of using a loop so that they can see which test fails?

Reply:
I added something to make a message for which test case failed, even if it's one in a loop.